Source/Core/Common/StringUtil.cpp

```diff
@@ -77,13 +77,10 @@ bool AsciiToHex(const std::string& _szValue, u32& result)
   // Set errno to a good state.
   errno = 0;
 
-  char* endptr = nullptr;
-  const u32 value = strtoul(_szValue.c_str(), &endptr, 16);
+  const u32 value = strtoul(_szValue.c_str(), nullptr, 16);
 
-  if (!endptr || *endptr)
-    return false;
-
-  if (errno == ERANGE)
+  // Fail if errno EINVAL or ERANGE
```

Comment:
That's true in C, not in C++ according to both [en.cppreference.com](http://en.cppreference.com/w/c/string/byte/strtoul) and [cplusplus](http://www.cplusplus.com/reference/cstdlib/strtoul/). The only errno value mentioned is ERANGE. I'm not against changing the if condition but the comment seems irrelevant. Do you have any concrete examples?